config/prometheus-sizing-env-base.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
```

Comment:
If the benchmarks are going to run inside a pod in k8s, do we actually have to have this? Can't we simply download kube-burner when we build the image?

Reply:
That is a great idea, I'll make sure we clone the repo in the Dockerfile. However, I'll keep the target `$(E2E_BENCHMARKING_ROOT)` so we clone the repo when running the benchmarks with `make run/benchmarks`, which is useful for local development. The target is not `PHONY`, so `make` will do nothing if the repo is already cloned, which is what will happen in the image